scipy/interpolate/_interpolate.py

```diff
@@ -125,8 +125,8 @@ class interp2d:
         `CloughTocher2DInterpolator`.
 
         For more details see
-        `https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff
-        <https://gist.github.com/ev-br/8544371b40f414b7eaf3fe6217209bff>`_
+        `https://scipy.github.io/devdocs/notebooks/interp_transition_guide.html
+        <https://scipy.github.io/devdocs/notebooks/interp_transition_guide.html>`_
```

Comment:
I remember for talk of using an inter sphinx link but given you can't actually click links here because of the theme I think the URL would be better.